frontend/src/bundles/common/components/sidebar/styles.module.scss

```diff
@@ -41,3 +46,129 @@ a {
         color: var(--white);
     }
 }
+
+// BURGER
+.burgerBackground {
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 10;
+    display: none;
+    width: var(--sidebar-width);
+    min-width: var(--sidebar-width);
+    max-width: var(--sidebar-width);
+    height: var(--header-height);
+    background-color: var(--white);
+    content: "";
+}
+
+.burgerWrapper {
+    position: fixed;
+    top: 0;
+    left: 0;
+    z-index: 30;
+    display: none;
+    width: var(--sidebar-width);
+    min-width: var(--sidebar-width);
+    max-width: var(--sidebar-width);
+    height: var(--header-height);
+}
+
+.burgerButton {
+    --size: 20px;
+    --color: var(--text-primary-dark);
```

Comment:
looks like you can use our common variable

Reply:
As for the "--color", this variable is not static, it changes from var(--text-primary-dark) to #ffd231 after this button is clicked
I don't know how to make this animation without --color